src/hooks/useRoom.ts

```diff
@@ -13,11 +14,13 @@ export const useRoom = (room: Room) => {
   const { user } = useUser();
   const userId = user?.uid;
 
+  const roomUrl = room.url;
+
   const venues = useSelector(orderedVenuesSelector);
 
   const roomVenue = useMemo(
-    () => venues?.find((venue) => room.url.endsWith(`/${venue.id}`)),
-    [room, venues]
+    () => venues?.find((venue) => roomUrl.endsWith(`/${venue.id}`)),
+    [roomUrl, venues]
   );
   const roomId = roomVenue ? roomVenue.name : room.url;
```

Comment:
Since this is used with `useRecentLocationUsers`, I still don't believe `room.url` is correct here. It should be the `venueName/roomName` slug.

Reply:
It is not related to slug/url lastSeenIn's logic

Reply:
I believe it may be?

```
const roomId = roomVenue ? roomVenue.name : room.url;
const { recentLocationUsers } = useRecentLocationUsers(roomId);
```

If we are trying to get `recentLocationUsers` with `room.url`, that isn't using the `venueName/roomName` slug, so will fail to retrieve them. Or so I would think? (I haven't looked deeply into the logic of `useRecentLocationUsers` to verify this yet)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1129/commits/53f57c40e24983470b7552d4ebed6f1715ff74ba